cmake/fake_tribits.cmake

```diff
@@ -102,6 +102,7 @@ FUNCTION(KOKKOS_ADD_TEST)
       NAME ${TEST_NAME}
       COMM serial mpi
       NUM_MPI_PROCS 1
+      ARGS ${TEST_ARGS}
```

Comment:
I don't think we use the name CMD_ARGS anywhere. We should probably just call it ARGS to match.